pkg/gui/filetree/file_manager.go

```diff
@@ -22,6 +31,7 @@ func NewFileManager(files []*models.File, log *logrus.Entry, showTree bool) *Fil
 		files:          files,
 		log:            log,
 		showTree:       showTree,
+		filter:         DisplayAll,
```

Comment:
Maybe make this configurable? Is it realistic that someone wants to only show staged/unstaged files on launch?